node-api/handlers/beacon/validators.go

```diff
@@ -95,7 +108,15 @@ func (h *Handler) GetStateValidatorBalances(c handlers.Context) (any, error) {
 		return nil, err
 	}
 	slot, err := utils.SlotFromStateID(req.StateID, h.backend)
+
+	// Check for error message "slot not found at state root"
 	if err != nil {
+		if strings.Contains(err.Error(), "slot not found at state root") {
```

Comment:
I know its not strictly in the scope of this PR but can we also make this error a strongly typed `ErrNoSlotForStateRoot` or so? And then this check can be done with `errors.Is(err, ErrNoSlotForStateRoot)`

Reply:
addressed.